er/groovy/src/main/java/org/linqs/psl/examples/er/Run.groovy

```diff
@@ -0,0 +1,302 @@
+package org.linqs.psl.examples.er;
+
+import org.linqs.psl.application.inference.MPEInference;
+import org.linqs.psl.application.learning.weight.VotedPerceptron;
```

Comment:
We use slightly different interfaces for inference and weight learning now.   
Inference - `org.linqs.psl.application.inference.InferenceApplication`
Weight Learning - `org.linqs.psl.application.learning.weight.WeightLearningApplication`

See current examples.